### PR TITLE
Improved deck summary component layout

### PR DESCRIPTION
**Pull request by piedpiper358**
Before:

After:

I understand that it may not be very correct to use regular expressions to truncate the string, but it can't be done through toLocaleDateString.

### CODE REVIEW

**Comment by piedpiper358** [comment]
Do the same thing for mercs team summary.

**Comment by piedpiper358** [comment]
Don't like these changes? Btw, they concern not only the Russian language.

**Comment by sebastientromp** [comment]
I added a comment in one of the files to better understand what is going on :)

And yeah, there are languages that take more room to say the same thing, but I'm not sure I want the app itself to handle all of the this (though accommodation can be made when it makes sense). I would rather that the translated strings themselves get shortened / abbreviated to fit inside the existing text.

**Comment by piedpiper358** [comment]
It seems to me that Firestone should stick to the general rule for localizable applications to leave about [30% extra space for other languages.](https://blog.alconost.com/en/how-to-prepare-a-game-for-localization) I think it can be done so that it doesn't look ugly in English.
For example, in this case, the lines in Russian are already shortened, but they still do not fit.

**Comment by sebastientromp** [comment]
I wasn't aware of that rule, thanks.
However it might be quite a big endeavor for existing components. 
I have renamed the English string to read "Used: " instead of "Last used: ", which should make the text smaller.
For the first line in Italian, I think an alternate translation should be found, since "Tasso di vittoria" is twice as long as "Winrate"

**Review by sebastientromp — CHANGES_REQUESTED** [review]

File: core/src/js/components/decktracker/main/decktracker-deck-summary.component.ts [hunk 1]
```diff
@@ -173,7 +173,8 @@ export class DecktrackerDeckSummaryComponent implements AfterViewInit {
 			month: 'short',
 			day: '2-digit',
 			year: 'numeric',
-		});
+		})
+		.replace(/\s+г\./, ""); //truncate date in russian
```
[the remark this answers is not in the document]
Comment (piedpiper358) [reply core/src/js/components/decktracker/main/decktracker-deck-summary.component.ts:177]: I googled it but the point is that toLocaleDateString function doesn't support anything like that. The year parameter can only be "numeric" - "2022 г." or "2-digit" - "22 г." Here is a page on [Stackoverflow](https://ru.stackoverflow.com/questions/816431/%D0%9A%D0%B0%D0%BA-%D1%83%D0%B1%D1%80%D0%B0%D1%82%D1%8C-%D0%93-%D0%B2-javsscript) with the same problem. We can instead make two calls to the toLocaleDateString function.

**Comment by sebastientromp** [comment]
Not a big fan of the solution, but we can use this as a first step at least :)

**Pull request merged**